src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractFeaturesEngine.java

```diff
@@ -234,6 +252,14 @@ private void processVQSRRecordForPosition(ExtractFeaturesRecord rec) {
         }
         builder.attribute(GATKVCFConstants.EXCESS_HET_KEY, String.format("%.3f", excessHetApprox));
 
+        if (rec.getDistinctAlleles() > 6) {
```

Comment:
is this a constant defined somewhere that we can use? (or should we make it one?)